modules/swagger-parser-v3/src/main/java/io/swagger/v3/parser/OpenAPIV3Parser.java

```diff
@@ -82,8 +86,13 @@ public OpenAPI read(String location, List<AuthorizationValue> auths, ParseOption
         OpenAPI output;
 
         List<SwaggerParserExtension> parserExtensions = getExtensions();
+	SwaggerParseResult parsed;
         for (SwaggerParserExtension extension : parserExtensions) {
-            output = extension.readLocation(location, auths,resolve).getOpenAPI();
+	    parsed = extension.readLocation(location, auths, resolve);
```

Comment:
This bit seems the most important since the other classes collect messages which are subsequently dropped.

Reply:
I think is missing the .getOpenAPI();

Reply:
That's below on L95, it was split out so the messages attached to the results could be output.